recipes/pymorton/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pymorton" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/trevorprater/{{ name }}/archive/refs/tags/1.0.5.tar.gz
+  sha256: f22ebe8c1f69ef4001c59f277dbe5403cb18f6bd3004ef19ac0e5fbdf5f865f6
+  patches:
+    - v1.0.7.patch
+    - fix-64-bit.patch
```

Comment:
Can you add a comment on why these patches are needed and/or if they are upstream?

Reply:
Thanks for the review, @ocefpaf, all done :)